docs/content/concepts/automation/schedules.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Schedules | Dagster Docs"
+description: "Use schedules to run your pipelines at fixed time intervals."
+---
+
+# Schedules
+
+Schedules are Dagster's way of supporting traditional methods of [automation](/concepts/automation), which allow you to specify when a [job](/concepts/ops-jobs-graphs/jobs) should run. Using schedules, you can define a fixed time interval to run a pipeline, such as daily, hourly, or Monday at 9:00 AM.
+
+Each interval of a schedule is called a **tick**, which is an indication that a job should begin executing. Ticks kick off **runs**, which is a single instance of a job being executed.
+
+When viewing a schedule in Dagster's UI, you can see the schedule's definition, executing timezone, targeted jobs and partitions, as well as its tick and run history.
+
+---
+
+## Benefits
+
+Using schedules helps you:
+
+- Predictably process and deliver data to stakeholders and business-critical applications
+- Consistently run data pipelines without the need for manual intervention
+- Optimize resource usage by scheduling pipelines to run during off-peak hours
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Cron syntax](https://en.wikipedia.org/wiki/Cron)
```

Comment:
Might be worth sharing here, but when dealing with Cron I almost always use https://crontab.guru/